docs/tutorial/part-three/index.md

```diff
@@ -135,11 +135,11 @@ export default () => (
 
 ![with-layout2](with-layout2.png)
 
-Sweet. You've installed and configured your very first Gatsby plugin!
+ आपने अपना पहला Gatsby प्लगइन इनस्टॉल और कॉन्फ़िगर किया है!
```

Comment:
`आपने अपना पहला Gatsby प्लगइन इनस्टॉल और कॉन्फ़िगर किया है!` => `बहुत बढ़िया आपने अपना पहला Gatsby प्लगइन इनस्टॉल और कॉन्फ़िगर किया है!`

Reply:
`बहुत बढ़िया` => `बहुत बढ़िया।`

missed purn viram.
sorry about this miss